### DISCUSSION TITLE
Welcome to wa-automate-nodejs Discussions!

### DISCUSSION THREAD

**Discussion by smashah** [open-wa/wa-automate-nodejs#2370] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexchristianqr** [comment]
Hola @smashah  tengo una consulta soy de Peru. Quiero pagar por una licencia pero quiero recibir información sobre esto:
![image](https://user-images.githubusercontent.com/22840027/184469139-b5ec831a-9fd5-4816-8fce-3e9836992b84.png)

¿Como funciona el numero de cuenta de host?
Confirmame si lo que entiendo es correcto. El numero de host es un identificador del servidor o computadora donde se estara ejecutando la aplicacion OPEN-WA nodejs.

Gracias.

**Reply by smashah** [reply]
The host account number is the number of the account that you use with open-wa (the account that you scan the qr code with). If you get the license and it is the wrong number then just email me the correct number and I will update it

**Reply by alexchristianqr** [reply]
Muchas gracias! por la información

**Comment by crescent1** [comment]
Hi @smashah, is it possible to create not only 1 host but many in one app? thank you,..

**Reply by smashah** [reply]
It's not advisable to have more than one session per node process. It is possible and there are some optimised orchestration tools/APIs for b2b clients.

**Reply by crescent1** [reply]
Thank you for the information.

**Comment by sai431** [comment]
HI @smashah  , can you please help me on information on pricing plan and charges for open-wa licensing ?
